File: server/iih_pub/iih.ci_pub/src/main/java/iih/ci/mr/nu/obstetrics/birthregis/d/BirthRegistrationDO.java
package iih.ci.mr.nu.obstetrics.birthregis.d;

import xap.sys.appfw.orm.desc.DescManager;
import xap.mw.coreitf.c.GlobalConst;
import xap.mw.coreitf.i.IDODesc;
import xap.mw.coreitf.d.*;
import xap.mw.core.data.*;
import iih.ci.mr.nu.obstetrics.birthregis.d.desc.BirthRegistrationDODesc;
import java.math.BigDecimal;

/**
 * 分娩登记薄 DO数据 
 * 
 */
public class BirthRegistrationDO extends BaseDO {
	private static final long serialVersionUID = 1L;

	public static final String ID_BIRTH_REG= "Id_birth_reg";
	public static final String ID_ORG= "Id_org";
	public static final String ID_GRP= "Id_grp";
	public static final String ID_ENT= "Id_ent";
	public static final String ID_PAT= "Id_pat";
	public static final String CODE_ENTP= "Code_entp";
	public static final String SORTNO= "Sortno";
	public static final String DT_DATE= "Dt_date";
	public static final String DT_TIME= "Dt_time";
	public static final String AGE= "Age";
	public static final String CODE_AMR_IP= "Code_amr_ip";
	public static final String NUM_PREG= "Num_preg";
	public static final String NUM_BIRTH= "Num_birth";
	public static final String WEEK_PREG= "Week_preg";
	public static final String ID_DELIVERY= "Id_delivery";
	public static final String SD_DELIVERY= "Sd_delivery";
	public static final String ID_POSITION= "Id_position";
	public static final String SD_POSITION= "Sd_position";
	public static final String NAME_OPER= "Name_oper";
	public static final String COMPLICATION= "Complication";
	public static final String NUM_BLEED_INTRAP= "Num_bleed_intrap";
	public static final String NUM_BLEED_POSTP= "Num_bleed_postp";
	public static final String RES_BLEED= "Res_bleed";
	public static final String ID_PERINEUM_STATUS= "Id_perineum_status";
	public static final String SD_PERINEUM_STATUS= "Sd_perineum_status";
	public static final String ID_SEX= "Id_sex";
	public static final String SD_SEX= "Sd_sex";
	public static final String WEIGHT= "Weight";
	public static final String HEIGHT= "Height";
	public static final String SITUATION_SPECIAL= "Situation_special";
	public static final String SCORE_MINUTE_ONE= "Score_minute_one";
	public static final String SCORE_MINUTE_FIVE= "Score_minute_five";
	public static final String SCORE_MINUTE_TEN= "Score_minute_ten";
	public static final String EU_ANMELDEN= "Eu_anmelden";
	public static final String EU_CONSULTHIV= "Eu_consulthiv";
	public static final String EU_FILTRATEHIV= "Eu_filtratehiv";
	public static final String ID_ASSAYTESTHIV= "Id_assaytesthiv";
	public static final String SD_ASSAYTESTHIV= "Sd_assaytesthiv";
	public static final String EU_SYPHILIS= "Eu_syphilis";
	public static final String EU_HBSAG= "Eu_hbsag";
	public static final String OTHER_SEXILLNESS= "Other_sexillness";
	public static final String NUM_FETUS= "Num_fetus";
	public static final String NUM_DEAD_FETUS= "Num_dead_fetus";
	public static final String NUM_DEAD_BIRTH= "Num_dead_birth";
	public static final String NUM_DEAD_NEWBORN= "Num_dead_newborn";
	public static final String DEFECTS_BIRTH= "Defects_birth";
	public static final String SUE_SEND= "Sue_send";
	public static final String ID_EMP_MIDWIVES= "Id_emp_midwives";
	public static final String ID_EMP_REG= "Id_emp_reg";
	public static final String CREATEDBY= "Createdby";
	public static final String CREATEDTIME= "Createdtime";
	public static final String MODIFIEDBY= "Modifiedby";
	public static final String MODIFIEDTIME= "Modifiedtime";
	public static final String NAME_PAT= "Name_pat";
	public static final String NAME_DELIVERY= "Name_delivery";
	public static final String NAME_POSITION= "Name_position";
	public static final String NAME_PERINEUM_STATUS= "Name_perineum_status";
	public static final String NAME_SEX= "Name_sex";
	public static final String NAME_ASSAYTESTHIV= "Name_assaytesthiv";
	public static final String NAME_EMP_MIDWIVES= "Name_emp_midwives";
	public static final String NAME_EMP_REG= "Name_emp_reg";
	public static final String DS = "Ds";
	public static final String SV = "Sv";
	
	public String getId_birth_reg() {
		return ((String) getAttrVal("Id_birth_reg"));
	}	
	public void setId_birth_reg(String Id_birth_reg) {
		setAttrVal("Id_birth_reg", Id_birth_reg);
	}
	public String getId_org() {
		return ((String) getAttrVal("Id_org"));
	}	
	public void setId_org(String Id_org) {
		setAttrVal("Id_org", Id_org);
	}
	public String getId_grp() {
		return ((String) getAttrVal("Id_grp"));
	}	
	public void setId_grp(String Id_grp) {
		setAttrVal("Id_grp", Id_grp);
	}
	public String getId_ent() {
		return ((String) getAttrVal("Id_ent"));
	}	
	public void setId_ent(String Id_ent) {
		setAttrVal("Id_ent", Id_ent);
	}
	public String getId_pat() {
		return ((String) getAttrVal("Id_pat"));
	}	
	public void setId_pat(String Id_pat) {
		setAttrVal("Id_pat", Id_pat);
	}
	public String getCode_entp() {
		return ((String) getAttrVal("Code_entp"));
	}	
	public void setCode_entp(String Code_entp) {
		setAttrVal("Code_entp", Code_entp);
	}
	public String getSortno() {
		return ((String) getAttrVal("Sortno"));
	}	
	public void setSortno(String Sortno) {
		setAttrVal("Sortno", Sortno);
	}
	public FDate getDt_date() {
		return ((FDate) getAttrVal("Dt_date"));
	}	
	public void setDt_date(FDate Dt_date) {
		setAttrVal("Dt_date", Dt_date);
	}
	public FTime getDt_time() {
		return ((FTime) getAttrVal("Dt_time"));
	}	
	public void setDt_time(FTime Dt_time) {
		setAttrVal("Dt_time", Dt_time);
	}
	public String getAge() {
		return ((String) getAttrVal("Age"));
	}	
	public void setAge(String Age) {
		setAttrVal("Age", Age);
	}
	public String getCode_amr_ip() {
		return ((String) getAttrVal("Code_amr_ip"));
	}	
	public void setCode_amr_ip(String Code_amr_ip) {
		setAttrVal("Code_amr_ip", Code_amr_ip);
	}
	public Integer getNum_preg() {
		return ((Integer) getAttrVal("Num_preg"));
	}	
	public void setNum_preg(Integer Num_preg) {
		setAttrVal("Num_preg", Num_preg);
	}
	public Integer getNum_birth() {
		return ((Integer) getAttrVal("Num_birth"));
	}	
	public void setNum_birth(Integer Num_birth) {
		setAttrVal("Num_birth", Num_birth);
	}
	public Integer getWeek_preg() {
		return ((Integer) getAttrVal("Week_preg"));
	}	
	public void setWeek_preg(Integer Week_preg) {
		setAttrVal("Week_preg", Week_preg);
	}
	public String getId_delivery() {
		return ((String) getAttrVal("Id_delivery"));
	}	
	public void setId_delivery(String Id_delivery) {
		setAttrVal("Id_delivery", Id_delivery);
	}
	public String getSd_delivery() {
		return ((String) getAttrVal("Sd_delivery"));
	}	
	public void setSd_delivery(String Sd_delivery) {
		setAttrVal("Sd_delivery", Sd_delivery);
	}
	public String getId_position() {
		return ((String) getAttrVal("Id_position"));
	}	
	public void setId_position(String Id_position) {
		setAttrVal("Id_position", Id_position);
	}
	public String getSd_position() {
		return ((String) getAttrVal("Sd_position"));
	}	
	public void setSd_position(String Sd_position) {
		setAttrVal("Sd_position", Sd_position);
	}
	public String getName_oper() {
		return ((String) getAttrVal("Name_oper"));
	}	
	public void setName_oper(String Name_oper) {
		setAttrVal("Name_oper", Name_oper);
	}
	public String getComplication() {
		return ((String) getAttrVal("Complication"));
	}	
	public void setComplication(String Complication) {
		setAttrVal("Complication", Complication);
	}
	public Integer getNum_bleed_intrap() {
		return ((Integer) getAttrVal("Num_bleed_intrap"));
	}	
	public void setNum_bleed_intrap(Integer Num_bleed_intrap) {
		setAttrVal("Num_bleed_intrap", Num_bleed_intrap);
	}
	public Integer getNum_bleed_postp() {
		return ((Integer) getAttrVal("Num_bleed_postp"));
	}	
	public void setNum_bleed_postp(Integer Num_bleed_postp) {
		setAttrVal("Num_bleed_postp", Num_bleed_postp);
	}
	public String getRes_bleed() {
		return ((String) getAttrVal("Res_bleed"));
	}	
	public void setRes_bleed(String Res_bleed) {
		setAttrVal("Res_bleed", Res_bleed);
	}
	public String getId_perineum_status() {
		return ((String) getAttrVal("Id_perineum_status"));
	}	
	public void setId_perineum_status(String Id_perineum_status) {
		setAttrVal("Id_perineum_status", Id_perineum_status);
	}
	public String getSd_perineum_status() {
		return ((String) getAttrVal("Sd_perineum_status"));
	}	
	public void setSd_perineum_status(String Sd_perineum_status) {
		setAttrVal("Sd_perineum_status", Sd_perineum_status);
	}
	public String getId_sex() {
		return ((String) getAttrVal("Id_sex"));
	}	
	public void setId_sex(String Id_sex) {
		setAttrVal("Id_sex", Id_sex);
	}
	public String getSd_sex() {
		return ((String) getAttrVal("Sd_sex"));
	}	
	public void setSd_sex(String Sd_sex) {
		setAttrVal("Sd_sex", Sd_sex);
	}
	public FDouble getWeight() {
		return ((FDouble) getAttrVal("Weight"));
	}	
	public void setWeight(FDouble Weight) {
		setAttrVal("Weight", Weight);
	}
	public Integer getHeight() {
		return ((Integer) getAttrVal("Height"));
	}	
	public void setHeight(Integer Height) {
		setAttrVal("Height", Height);
	}
	public String getSituation_special() {
		return ((String) getAttrVal("Situation_special"));
	}	
	public void setSituation_special(String Situation_special) {
		setAttrVal("Situation_special", Situation_special);
	}
	public Integer getScore_minute_one() {
		return ((Integer) getAttrVal("Score_minute_one"));
	}	
	public void setScore_minute_one(Integer Score_minute_one) {
		setAttrVal("Score_minute_one", Score_minute_one);
	}
	public Integer getScore_minute_five() {
		return ((Integer) getAttrVal("Score_minute_five"));
	}	
	public void setScore_minute_five(Integer Score_minute_five) {
		setAttrVal("Score_minute_five", Score_minute_five);
	}
	public Integer getScore_minute_ten() {
		return ((Integer) getAttrVal("Score_minute_ten"));
	}	
	public void setScore_minute_ten(Integer Score_minute_ten) {
		setAttrVal("Score_minute_ten", Score_minute_ten);
	}
	public Integer getEu_anmelden() {
		return ((Integer) getAttrVal("Eu_anmelden"));
	}	
	public void setEu_anmelden(Integer Eu_anmelden) {
		setAttrVal("Eu_anmelden", Eu_anmelden);
	}
	public Integer getEu_consulthiv() {
		return ((Integer) getAttrVal("Eu_consulthiv"));
	}	
	public void setEu_consulthiv(Integer Eu_consulthiv) {
		setAttrVal("Eu_consulthiv", Eu_consulthiv);
	}
	public Integer getEu_filtratehiv() {
		return ((Integer) getAttrVal("Eu_filtratehiv"));
	}	
	public void setEu_filtratehiv(Integer Eu_filtratehiv) {
		setAttrVal("Eu_filtratehiv", Eu_filtratehiv);
	}
	public String getId_assaytesthiv() {
		return ((String) getAttrVal("Id_assaytesthiv"));
	}	
	public void setId_assaytesthiv(String Id_assaytesthiv) {
		setAttrVal("Id_assaytesthiv", Id_assaytesthiv);
	}
	public String getSd_assaytesthiv() {
		return ((String) getAttrVal("Sd_assaytesthiv"));
	}	
	public void setSd_assaytesthiv(String Sd_assaytesthiv) {
		setAttrVal("Sd_assaytesthiv", Sd_assaytesthiv);
	}
	public Integer getEu_syphilis() {
		return ((Integer) getAttrVal("Eu_syphilis"));
	}	
	public void setEu_syphilis(Integer Eu_syphilis) {
		setAttrVal("Eu_syphilis", Eu_syphilis);
	}
	public Integer getEu_hbsag() {
		return ((Integer) getAttrVal("Eu_hbsag"));
	}	
	public void setEu_hbsag(Integer Eu_hbsag) {
		setAttrVal("Eu_hbsag", Eu_hbsag);
	}
	public String getOther_sexillness() {
		return ((String) getAttrVal("Other_sexillness"));
	}	
	public void setOther_sexillness(String Other_sexillness) {
		setAttrVal("Other_sexillness", Other_sexillness);
	}
	public Integer getNum_fetus() {
		return ((Integer) getAttrVal("Num_fetus"));
	}	
	public void setNum_fetus(Integer Num_fetus) {
		setAttrVal("Num_fetus", Num_fetus);
	}
	public Integer getNum_dead_fetus() {
		return ((Integer) getAttrVal("Num_dead_fetus"));
	}	
	public void setNum_dead_fetus(Integer Num_dead_fetus) {
		setAttrVal("Num_dead_fetus", Num_dead_fetus);
	}
	public Integer getNum_dead_birth() {
		return ((Integer) getAttrVal("Num_dead_birth"));
	}	
	public void setNum_dead_birth(Integer Num_dead_birth) {
		setAttrVal("Num_dead_birth", Num_dead_birth);
	}
	public Integer getNum_dead_newborn() {
		return ((Integer) getAttrVal("Num_dead_newborn"));
	}	
	public void setNum_dead_newborn(Integer Num_dead_newborn) {
		setAttrVal("Num_dead_newborn", Num_dead_newborn);
	}
	public String getDefects_birth() {
		return ((String) getAttrVal("Defects_birth"));
	}	
	public void setDefects_birth(String Defects_birth) {
		setAttrVal("Defects_birth", Defects_birth);
	}
	public String getSue_send() {
		return ((String) getAttrVal("Sue_send"));
	}	
	public void setSue_send(String Sue_send) {
		setAttrVal("Sue_send", Sue_send);
	}
	public String getId_emp_midwives() {
		return ((String) getAttrVal("Id_emp_midwives"));
	}	
	public void setId_emp_midwives(String Id_emp_midwives) {
		setAttrVal("Id_emp_midwives", Id_emp_midwives);
	}
	public String getId_emp_reg() {
		return ((String) getAttrVal("Id_emp_reg"));
	}	
	public void setId_emp_reg(String Id_emp_reg) {
		setAttrVal("Id_emp_reg", Id_emp_reg);
	}
	public String getCreatedby() {
		return ((String) getAttrVal("Createdby"));
	}	
	public void setCreatedby(String Createdby) {
		setAttrVal("Createdby", Createdby);
	}
	public FDateTime getCreatedtime() {
		return ((FDateTime) getAttrVal("Createdtime"));
	}	
	public void setCreatedtime(FDateTime Createdtime) {
		setAttrVal("Createdtime", Createdtime);
	}
	public String getModifiedby() {
		return ((String) getAttrVal("Modifiedby"));
	}	
	public void setModifiedby(String Modifiedby) {
		setAttrVal("Modifiedby", Modifiedby);
	}
	public FDateTime getModifiedtime() {
		return ((FDateTime) getAttrVal("Modifiedtime"));
	}	
	public void setModifiedtime(FDateTime Modifiedtime) {
		setAttrVal("Modifiedtime", Modifiedtime);
	}
	public String getName_pat() {
		return ((String) getAttrVal("Name_pat"));
	}	
	public void setName_pat(String Name_pat) {
		setAttrVal("Name_pat", Name_pat);
	}
	public String getName_delivery() {
		return ((String) getAttrVal("Name_delivery"));
	}	
	public void setName_delivery(String Name_delivery) {
		setAttrVal("Name_delivery", Name_delivery);
	}
	public String getName_position() {
		return ((String) getAttrVal("Name_position"));
	}	
	public void setName_position(String Name_position) {
		setAttrVal("Name_position", Name_position);
	}
	public String getName_perineum_status() {
		return ((String) getAttrVal("Name_perineum_status"));
	}	
	public void setName_perineum_status(String Name_perineum_status) {
		setAttrVal("Name_perineum_status", Name_perineum_status);
	}
	public String getName_sex() {
		return ((String) getAttrVal("Name_sex"));
	}	
	public void setName_sex(String Name_sex) {
		setAttrVal("Name_sex", Name_sex);
	}
	public String getName_assaytesthiv() {
		return ((String) getAttrVal("Name_assaytesthiv"));
	}	
	public void setName_assaytesthiv(String Name_assaytesthiv) {
		setAttrVal("Name_assaytesthiv", Name_assaytesthiv);
	}
	public String getName_emp_midwives() {
		return ((String) getAttrVal("Name_emp_midwives"));
	}	
	public void setName_emp_midwives(String Name_emp_midwives) {
		setAttrVal("Name_emp_midwives", Name_emp_midwives);
	}
	public String getName_emp_reg() {
		return ((String) getAttrVal("Name_emp_reg"));
	}	
	public void setName_emp_reg(String Name_emp_reg) {
		setAttrVal("Name_emp_reg", Name_emp_reg);
	}

	public Integer getDs() {
		return ((Integer) getAttrVal(GlobalConst.DELETE_SIGN));
	}
	
	public void setDs(Integer ds) {
		setAttrVal(GlobalConst.DELETE_SIGN, ds);
	}
	
	public FDateTime getSv() {
		return ((FDateTime) getAttrVal(GlobalConst.SYSTEM_VERSION));
	}
	
	public void setSv(FDateTime sv) {
		setAttrVal(GlobalConst.SYSTEM_VERSION, sv);
	}

//	@Override
//	public java.lang.String getParentPKFieldName() {
//		return null;
//	}
  
	@Override
	public String getPKFieldName() {
		return "Id_birth_reg";
	}
	
	@Override
	public String getTableName() {	  
		return "CI_MR_NU_BIRTH_REG";
	}
	
	@Override
	public IDODesc getDODesc() {
		return DescManager.getInstance().getDODesc(BirthRegistrationDODesc.class);
	}
	
}

File: client/iih_stub/iih.ci_stub/iih.ci.mr/nu/obstetrics/birthregis/d/BirthRegistrationDO.cs

using System;
using xap.mw.core.data;
using xap.mw.coreitf.d;
using System.Collections.Generic;

namespace iih.ci.mr.nu.obstetrics.birthregis.d
{
    /// <summary>
    /// BirthRegistrationDO 的摘要说明。
    /// </summary>
    public class BirthRegistrationDO : BaseDO {

		public const string TABLE_NAME = "CI_MR_NU_BIRTH_REG";
		public const string TABLE_ALIAS_NAME = "a0";

        public BirthRegistrationDO() {
        }
		public string Id_birth_reg {
            get { return getAttrVal<string>("Id_birth_reg",null); }
            set { setAttrVal<string>("Id_birth_reg", value); }
        }
		public string Id_org {
            get { return getAttrVal<string>("Id_org",null); }
            set { setAttrVal<string>("Id_org", value); }
        }
		public string Id_grp {
            get { return getAttrVal<string>("Id_grp",null); }
            set { setAttrVal<string>("Id_grp", value); }
        }
		public string Id_ent {
            get { return getAttrVal<string>("Id_ent",null); }
            set { setAttrVal<string>("Id_ent", value); }
        }
		public string Id_pat {
            get { return getAttrVal<string>("Id_pat",null); }
            set { setAttrVal<string>("Id_pat", value); }
        }
		public string Code_entp {
            get { return getAttrVal<string>("Code_entp",null); }
            set { setAttrVal<string>("Code_entp", value); }
        }
		public string Sortno {
            get { return getAttrVal<string>("Sortno",null); }
            set { setAttrVal<string>("Sortno", value); }
        }
		public DateTime? Dt_date {
            get { return getAttrVal<FDate>("Dt_date",null); }
            set { setAttrVal<FDate>("Dt_date", value); }
        }
		public DateTime? Dt_time {
            get { return getAttrVal<FTime>("Dt_time",null); }
            set { setAttrVal<FTime>("Dt_time", value); }
        }
		public string Age {
            get { return getAttrVal<string>("Age",null); }
            set { setAttrVal<string>("Age", value); }
        }
		public string Code_amr_ip {
            get { return getAttrVal<string>("Code_amr_ip",null); }
            set { setAttrVal<string>("Code_amr_ip", value); }
        }
		public int? Num_preg {
            get { return getAttrVal<int?>("Num_preg",null); }
            set { setAttrVal<int?>("Num_preg", value); }
        }
		public int? Num_birth {
            get { return getAttrVal<int?>("Num_birth",null); }
            set { setAttrVal<int?>("Num_birth", value); }
        }
		public int? Week_preg {
            get { return getAttrVal<int?>("Week_preg",null); }
            set { setAttrVal<int?>("Week_preg", value); }
        }
		public string Id_delivery {
            get { return getAttrVal<string>("Id_delivery",null); }
            set { setAttrVal<string>("Id_delivery", value); }
        }
		public string Sd_delivery {
            get { return getAttrVal<string>("Sd_delivery",null); }
            set { setAttrVal<string>("Sd_delivery", value); }
        }
		public string Id_position {
            get { return getAttrVal<string>("Id_position",null); }
            set { setAttrVal<string>("Id_position", value); }
        }
		public string Sd_position {
            get { return getAttrVal<string>("Sd_position",null); }
            set { setAttrVal<string>("Sd_position", value); }
        }
		public string Name_oper {
            get { return getAttrVal<string>("Name_oper",null); }
            set { setAttrVal<string>("Name_oper", value); }
        }
		public string Complication {
            get { return getAttrVal<string>("Complication",null); }
            set { setAttrVal<string>("Complication", value); }
        }
		public int? Num_bleed_intrap {
            get { return getAttrVal<int?>("Num_bleed_intrap",null); }
            set { setAttrVal<int?>("Num_bleed_intrap", value); }
        }
		public int? Num_bleed_postp {
            get { return getAttrVal<int?>("Num_bleed_postp",null); }
            set { setAttrVal<int?>("Num_bleed_postp", value); }
        }
		public string Res_bleed {
            get { return getAttrVal<string>("Res_bleed",null); }
            set { setAttrVal<string>("Res_bleed", value); }
        }
		public string Id_perineum_status {
            get { return getAttrVal<string>("Id_perineum_status",null); }
            set { setAttrVal<string>("Id_perineum_status", value); }
        }
		public string Sd_perineum_status {
            get { return getAttrVal<string>("Sd_perineum_status",null); }
            set { setAttrVal<string>("Sd_perineum_status", value); }
        }
		public string Id_sex {
            get { return getAttrVal<string>("Id_sex",null); }
            set { setAttrVal<string>("Id_sex", value); }
        }
		public string Sd_sex {
            get { return getAttrVal<string>("Sd_sex",null); }
            set { setAttrVal<string>("Sd_sex", value); }
        }
		public FDouble Weight {
            get { return getAttrVal<FDouble>("Weight",null); }
            set { setAttrVal<FDouble>("Weight", value); }
        }
		public int? Height {
            get { return getAttrVal<int?>("Height",null); }
            set { setAttrVal<int?>("Height", value); }
        }
		public string Situation_special {
            get { return getAttrVal<string>("Situation_special",null); }
            set { setAttrVal<string>("Situation_special", value); }
        }
		public int? Score_minute_one {
            get { return getAttrVal<int?>("Score_minute_one",null); }
            set { setAttrVal<int?>("Score_minute_one", value); }
        }
		public int? Score_minute_five {
            get { return getAttrVal<int?>("Score_minute_five",null); }
            set { setAttrVal<int?>("Score_minute_five", value); }
        }
		public int? Score_minute_ten {
            get { return getAttrVal<int?>("Score_minute_ten",null); }
            set { setAttrVal<int?>("Score_minute_ten", value); }
        }
		public int? Eu_anmelden {
            get { return getAttrVal<int?>("Eu_anmelden",null); }
            set { setAttrVal<int?>("Eu_anmelden", value); }
        }
		public int? Eu_consulthiv {
            get { return getAttrVal<int?>("Eu_consulthiv",null); }
            set { setAttrVal<int?>("Eu_consulthiv", value); }
        }
		public int? Eu_filtratehiv {
            get { return getAttrVal<int?>("Eu_filtratehiv",null); }
            set { setAttrVal<int?>("Eu_filtratehiv", value); }
        }
		public string Id_assaytesthiv {
            get { return getAttrVal<string>("Id_assaytesthiv",null); }
            set { setAttrVal<string>("Id_assaytesthiv", value); }
        }
		public string Sd_assaytesthiv {
            get { return getAttrVal<string>("Sd_assaytesthiv",null); }
            set { setAttrVal<string>("Sd_assaytesthiv", value); }
        }
		public int? Eu_syphilis {
            get { return getAttrVal<int?>("Eu_syphilis",null); }
            set { setAttrVal<int?>("Eu_syphilis", value); }
        }
		public int? Eu_hbsag {
            get { return getAttrVal<int?>("Eu_hbsag",null); }
            set { setAttrVal<int?>("Eu_hbsag", value); }
        }
		public string Other_sexillness {
            get { return getAttrVal<string>("Other_sexillness",null); }
            set { setAttrVal<string>("Other_sexillness", value); }
        }
		public int? Num_fetus {
            get { return getAttrVal<int?>("Num_fetus",null); }
            set { setAttrVal<int?>("Num_fetus", value); }
        }
		public int? Num_dead_fetus {
            get { return getAttrVal<int?>("Num_dead_fetus",null); }
            set { setAttrVal<int?>("Num_dead_fetus", value); }
        }
		public int? Num_dead_birth {
            get { return getAttrVal<int?>("Num_dead_birth",null); }
            set { setAttrVal<int?>("Num_dead_birth", value); }
        }
		public int? Num_dead_newborn {
            get { return getAttrVal<int?>("Num_dead_newborn",null); }
            set { setAttrVal<int?>("Num_dead_newborn", value); }
        }
		public string Defects_birth {
            get { return getAttrVal<string>("Defects_birth",null); }
            set { setAttrVal<string>("Defects_birth", value); }
        }
		public string Sue_send {
            get { return getAttrVal<string>("Sue_send",null); }
            set { setAttrVal<string>("Sue_send", value); }
        }
		public string Id_emp_midwives {
            get { return getAttrVal<string>("Id_emp_midwives",null); }
            set { setAttrVal<string>("Id_emp_midwives", value); }
        }
		public string Id_emp_reg {
            get { return getAttrVal<string>("Id_emp_reg",null); }
            set { setAttrVal<string>("Id_emp_reg", value); }
        }
		public string Createdby {
            get { return getAttrVal<string>("Createdby",null); }
            set { setAttrVal<string>("Createdby", value); }
        }
		public DateTime? Createdtime {
            get { return getAttrVal<FDateTime>("Createdtime",null); }
            set { setAttrVal<FDateTime>("Createdtime", value); }
        }
		public string Modifiedby {
            get { return getAttrVal<string>("Modifiedby",null); }
            set { setAttrVal<string>("Modifiedby", value); }
        }
		public DateTime? Modifiedtime {
            get { return getAttrVal<FDateTime>("Modifiedtime",null); }
            set { setAttrVal<FDateTime>("Modifiedtime", value); }
        }
		public string Name_pat {
            get { return getAttrVal<string>("Name_pat",null); }
            set { setAttrVal<string>("Name_pat", value); }
        }
		public string Name_delivery {
            get { return getAttrVal<string>("Name_delivery",null); }
            set { setAttrVal<string>("Name_delivery", value); }
        }
		public string Name_position {
            get { return getAttrVal<string>("Name_position",null); }
            set { setAttrVal<string>("Name_position", value); }
        }
		public string Name_perineum_status {
            get { return getAttrVal<string>("Name_perineum_status",null); }
            set { setAttrVal<string>("Name_perineum_status", value); }
        }
		public string Name_sex {
            get { return getAttrVal<string>("Name_sex",null); }
            set { setAttrVal<string>("Name_sex", value); }
        }
		public string Name_assaytesthiv {
            get { return getAttrVal<string>("Name_assaytesthiv",null); }
            set { setAttrVal<string>("Name_assaytesthiv", value); }
        }
		public string Name_emp_midwives {
            get { return getAttrVal<string>("Name_emp_midwives",null); }
            set { setAttrVal<string>("Name_emp_midwives", value); }
        }
		public string Name_emp_reg {
            get { return getAttrVal<string>("Name_emp_reg",null); }
            set { setAttrVal<string>("Name_emp_reg", value); }
        }
        public int Ds {
            get { return getAttrVal<int>("Ds",0);}
            set { setAttrVal<int>("Ds", value); }
        }

        public DateTime? Sv        {
            get { return getAttrVal<FDateTime>("Sv",null); }
            set { setAttrVal<FDateTime>("Sv", value); }
        }
        
        /// <summary>
        /// 返回表名
        /// </summary>
        /// <returns></returns>
        public override string getTableName()
        {
            return "CI_MR_NU_BIRTH_REG";
        }
        
        /// <summary>
        /// 返回表别名
        /// </summary>
        /// <returns></returns>
        public override string getAliasTableName()
        {
            return "a0";
        }
        
        /// <summary>
        /// 返回主键字段名
        /// </summary>
        /// <returns></returns>
        public override string getPKFieldName()
        {
            return "Id_birth_reg";
        }

        /// <summary>
        /// 返回全路径DO类名
        /// </summary>
        /// <returns></returns>
        public override string getFullClassName()
        {
            return "iih.ci.mr.nu.obstetrics.birthregis.d.BirthRegistrationDO";
        }

        /// <summary>
        /// 返回属性列表
        /// </summary>
        /// <returns></returns>
        public override string[] getAttrNames()
        {
            return new string[]{"Ds","Sv",  "Id_birth_reg", "Id_org", "Id_grp", "Id_ent", "Id_pat", "Code_entp", "Sortno", "Dt_date", "Dt_time", "Age", "Code_amr_ip", "Num_preg", "Num_birth", "Week_preg", "Id_delivery", "Sd_delivery", "Id_position", "Sd_position", "Name_oper", "Complication", "Num_bleed_intrap", "Num_bleed_postp", "Res_bleed", "Id_perineum_status", "Sd_perineum_status", "Id_sex", "Sd_sex", "Weight", "Height", "Situation_special", "Score_minute_one", "Score_minute_five", "Score_minute_ten", "Eu_anmelden", "Eu_consulthiv", "Eu_filtratehiv", "Id_assaytesthiv", "Sd_assaytesthiv", "Eu_syphilis", "Eu_hbsag", "Other_sexillness", "Num_fetus", "Num_dead_fetus", "Num_dead_birth", "Num_dead_newborn", "Defects_birth", "Sue_send", "Id_emp_midwives", "Id_emp_reg", "Createdby", "Createdtime", "Modifiedby", "Modifiedtime", "Name_pat", "Name_delivery", "Name_position", "Name_perineum_status", "Name_sex", "Name_assaytesthiv", "Name_emp_midwives", "Name_emp_reg"};
        }
        
    }
}
